### PR TITLE
Issue 39945: match related text in change summary in tests

### PR DESCRIPTION
**Pull request by labkey-ankurj**

### CODE REVIEW

File: src/org/labkey/test/WebDriverWrapper.java [hunk 1]
```diff
@@ -1458,7 +1458,7 @@ public void assertTextPresent(TextSearcher searcher, String... texts)
 
         if (!missingTexts.isEmpty())
         {
-            String failMsg = (missingTexts.size() == 1 ? "Text '" : "Texts ['") + String.join("', '", missingTexts) + "'";
+            String failMsg = (missingTexts.size() == 1 ? "Text '" : "Texts ['") + String.join("','", missingTexts) + "'";
```
Comment (labkey-jeckels) [inline src/org/labkey/test/WebDriverWrapper.java]: Is this change necessary?

Comment (labkey-ankurj) [reply src/org/labkey/test/WebDriverWrapper.java]: It's not.. Just keeping the fail msg consistent with the text.

Comment (labkey-ankurj) [reply src/org/labkey/test/WebDriverWrapper.java]: I see, this is used by many places. I will update.

**Review by labkey-jeckels — APPROVED** [review]
Just saw Ankur's update - I'm approving this change since it's now just targeted to the issues test, but @labkey-sweta  and @labkey-tchad please let us know if you have concerns

**Review by labkey-tchad — APPROVED** [review]

**Review by labkey-tchad — CHANGES_REQUESTED** [review]
This is just causing the test to fail: https://teamcity.labkey.org/viewLog.html?buildId=939541&tab=buildResultsDiv&buildTypeId=LabKey_203Release_Community_ModuleSuites_IssuesSqlserver

**Comment by labkey-ankurj** [comment]
@labkey-tchad Yes, I reversed the order of related matching text.

**Review by labkey-tchad — APPROVED** [review]
Didn't realize this was verifying the comment text. Maybe add a comment to that effect.

**Comment by labkey-ankurj** [comment]
@labkey-tchad added
